settings.gradle.kts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+pluginManagement {
```

Comment:
Why is this needed?  It seems an odd place for it and it's not in `settings.gradle`

Reply:
Its a slow move to modernization: https://docs.gradle.org/current/userguide/plugins.html#sec:binary_plugin_locations

It currently get duplicated by `build.gradle` and the`buildscript` block - but eventually that will be removed as the project is modernized.